Update CreateWeaponObject.md

jonassvensson4 opened the pull request:
I added some examples to make it more clear for people that it's required to use the RequestWeaponAsset native before using this one. I also changed the heading param to size since it's clearly increasing the size of the weapon and not changing the heading. 

Had this set to 8.0, you could imagine what 360 "degrees" would look like ;)
https://i.imgur.com/FKPt52J.png

smallo92 commented:
If you are showing people how to use it, you should tell them the right way ;)
```lua
local weapon = `WEAPON_ASSAULTRIFLE`
RequestWeaponAsset(weapon)
while not HasWeaponAssetLoaded(weapon) do
    Wait(0)
end

local coords = GetEntityCoords(PlayerPedId())
CreateWeaponObject(weapon, 0, coords.xyz)
```

jonassvensson4 commented:
> If you are showing people how to use it, you should tell them the right way ;)
> 
> ```lua
> local weapon = `WEAPON_ASSAULTRIFLE`
> RequestWeaponAsset(weapon)
> while not HasWeaponAssetLoaded(weapon) do
>     Wait(0)
> end
> 
> local coords = GetEntityCoords(PlayerPedId())
> CreateWeaponObject(weapon, 0, coords.xyz)
> ```

You're right, it was wrong of me to assume people would understand that :) I wanted to keep it small and compact. I've added your example and fixed the javascript example

sgottelk commented on WEAPON/CreateWeaponObject.md:
````diff
@@ -17,14 +17,52 @@ NativeDB Added Parameter 9: Any p8
 NativeDB Added Parameter 10: Any p9
 ```
 
+```
+To be able to create a weapon object you'll need to request the weapon asset with the REQUEST_WEAPON_ASSET native
+```
````
code fence here is unnecessary

jonassvensson4 commented:
Alright I removed the code fence. I assume you didn't mean that the entire comment was uneccessary. I can remove it of course, I just think it's good to be overclear about it.

technetium-cfx commented on WEAPON/CreateWeaponObject.md:
```diff
@@ -17,14 +17,50 @@ NativeDB Added Parameter 9: Any p8
+
+local coords = GetEntityCoords(PlayerPedId())
+
+CreateWeaponObject(weapon, 0, coords.xyz)
```
`.xyx` is not necessary.
